script/metric/extract_conflict_mineral_report.rb

```diff
@@ -0,0 +1,116 @@
+require File.expand_path('../../../config/environment',  __FILE__)
+require 'colorize'
+require 'json'
+require 'byebug'
+require 'rubygems'
+require 'csv'
+
+file_name = 'script/metric/data/certh.json'
+
+text = File.open(file_name).read
+
+def generate_aliases_hash
+  aliases_hash = {}
+  aliases_cards = Card.search right: 'aliases',
+                              left: { type_id: Card::WikirateCompanyID }
+  aliases_cards.each do |aliases_card|
+    aliases_card.item_names.each do |name|
+      aliases_hash[name.downcase] = aliases_card.cardname.left
+    end
+  end
+  aliases_hash
+end
```

Comment:
Isn't this reusable?  Seems like it should be in a shared library somewhere, no?